### PR TITLE
Init blog with vuejs

### PR DESCRIPTION
**Pull request by kiranparajuli589**
- Init blog with VueJS infrastructure
- Static markdown pages blog

### CODE REVIEW

**Review by saw-jan — COMMENTED** [review]
Lint fail vayexa

**Comment by kiranparajuli589** [comment]
> Lint fail vayexa

fixed :rocket:

File: README.md [hunk 1]
```diff
@@ -1 +1,57 @@
-blog posts of JankariTech: https://blog.jankaritech.com
+# A Blogging Platform
+
+A beautiful blogging platform for the JankariTech peoples.
+
+## Installation
+1. Requirements
+   - [NodeJS (<16.0.0)](https://nodejs.org/en/download/)
+   - [pnpm (<7.0.0)](https://pnpm.io/installation)
```
Comment (grgprarup) [inline README.md:8]:
```suggestion
   - [NodeJS (>16.0.0)](https://nodejs.org/en/download/)
   - [pnpm (>7.0.0)](https://pnpm.io/installation)
```

Comment (grgprarup) [inline README.md:8]:
```suggestion
   - [NodeJS (>16.0.0)](https://nodejs.org/en/download/)
   - [pnpm (>7.0.0)](https://pnpm.io/installation)
```

**Review by grgprarup — APPROVED** [review]
LGTM :+1:

**Pull request merged**